katas/content/oracles/index.md

```diff
@@ -58,7 +58,7 @@ An oracle in the quantum world is a "black box" operation that is used as input
 Many quantum algorithms assume an oracle implementation of some classical function as input, but this is a very strong assumption - sometimes implementing the oracle for a function is a lot more complex than the algorithm that will use this oracle!  
 In this kata, you will learn the properties of quantum oracles and how to implement them.
 
-A quantum oracle implements a function $f: \{0,1\}^n \rightarrow \{0,1\}^m$, where $x$ is an $n$-bit input state of the form $x = (x_{0}, x_{1}, \dots, x_{n-1})$. In most commonly used cases $m=1$, i.e., the function can return values $0$ or $1$; in this kata, we will focus on this class of functions.
+A quantum oracle implements a function $f(x): \{0,1\}^n \rightarrow \{0,1\}^m$, where $x$ is an $n$-bit input state of the form $x = (x_{0}, x_{1}, \dots, x_{n-1})$. In most commonly used cases $m=1$, that is, the function can return values $0$ or $1$. In this kata, we will focus on this class of functions.
```

Comment:
f(x) would be mathematically incorrect here. It should be f.

Reply:
Should we then change the second part to be something like the following?

> where the input is $n$-bits of the form $(x_{0}, x_{1}, \dots, x_{n-1})$

Reply:
Maybe "where the input is the n-bit state of the form..."?

Reply:
Updated.